File: Tag6/Tag6.jl
using DelimitedFiles

input = readdlm("Tag6/Tag5_daten_test.txt", ',', Int64, '\n');

fishes = zeros(Int64, 9)
a = zeros(Int64, 9)
for element in input
    fishes[element+1] += 1
end

steps = 80
for i in 1:80
    b = circshift(fishes, -1)
    
    fishes[7] += fishes[end]
end
a=1
print(sum(fishes))

File: Tag6/Tag6.py
import numpy as np
# data import
data = np.genfromtxt("Tag6/Tag5_daten.txt", delimiter=',', dtype=int)

fishes = np.zeros((9,1))

for i in range(9):
    for element in data:
        if element == i:
            fishes[i] += 1

# calculations
steps = 256
for i in range(steps):
    fishes = np.roll(fishes, -1)
    fishes[6] += fishes[-1]


print(np.sum(fishes))
